### DISCUSSION TITLE
Is it possible to use Spark Connect with stackable Spark operator?

### DISCUSSION THREAD

**Discussion by paulpaul1076** [stackabletech/spark-k8s-operator#351] [Q&A] [answered]
There's a new feature since Spark 3.5.0, called Spark Connect, can we use it with stackable Spark operator?

**Comment by razvan** [comment, marked as answer]
Not yet. You can follow this [issue](https://github.com/stackabletech/spark-k8s-operator/issues/284) for updates.
